lib/plug/static.ex

```diff
@@ -240,11 +237,11 @@ defmodule Plug.Static do
     {size, mtime} |> :erlang.phash2() |> Integer.to_string(16)
   end
 
-  defp file_encoding(conn, path, gzip, brotli) do
+  defp file_encoding(conn, path, gzip?, brotli?) do
     cond do
-      file_info = brotli && accept_encoding?(conn, "br") && regular_file_info(path <> ".br") ->
+      file_info = brotli? and accept_encoding?(conn, "br") and regular_file_info(path <> ".br") ->
         {:ok, put_resp_header(conn, "content-encoding", "br"), file_info, path <> ".br"}
-      file_info = gzip && accept_encoding?(conn, "gzip") && regular_file_info(path <> ".gz") ->
+      file_info = gzip? and accept_encoding?(conn, "gzip") and regular_file_info(path <> ".gz") ->
```

Comment:
I don't like to rely on the fact that `and` does not check its second args. So I would use `&&` because it is clearer for folks reading the code.